src/geometry.h

```diff
@@ -482,7 +482,8 @@ Point anyCenter(iterator begin, iterator end, BoundaryFunction apply_boundary, c
         apply_boundary(center);
         return center;
     } else {
-        throw std::runtime_error("cannot calculate center with zero weights");
+        std::cerr << "warning: sum of weights is 0! setting center to (0,0,0)" << std::endl;
```

Comment:
use `faunus_logger->warn("...");` instead of `cerr`

Reply:
Thanks for reviewing the PR, @mlund! It is a workaround I've been using to simulate an periodic chain by connecting the termini across the periodic boundaries.